### ISSUE TITLE
goplay.be: ads

### ISSUE THREAD

**Issue by Maiskolf** [uBlockOrigin/uAssets#17314]
### Prerequisites

- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I forced an update of my filter lists. (Click the ["Purge all caches"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches) button while holding the 'Shift' key, then click the ["Update now"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now) button.)
- [X] I did not remove any of the [default filter lists](https://user-images.githubusercontent.com/585534/91296476-ab9d8c00-e76a-11ea-9fb9-87d6861bdf73.png), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable external filter lists, or I have verified that the issue still occurs without enabling external filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin (uBO) along with other content blocker extensions.
- [X] I have verified that the web browser's built-in blocker or [DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking) (standalone or through a VPN) is not causing the issue.
- [ ] I did not answer truthfully to **ALL** the above checkpoints.

### URL address of the web page

`https://www.goplay.be/video/de-mol/de-mol-s11/de-mol-s11-aflevering-2`

### Category

ads

### Description

Ads are blocked correctly, but the video player shows an ad layover for the time that the ads would normally play. This layover blocks player controls (e.g. you cannot skip forward or backwards) and also pauses the video when the video player is not in focus (e.g. when switching tabs). When starting the video, the layover shows for 0:45 seconds but this increases to 3:00 minutes during future ad breaks. See screenshot attached.

Related issue: https://github.com/uBlockOrigin/uAssets/issues/16881

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

![image](https://user-images.githubusercontent.com/96501433/228170774-a6806782-5806-45d1-b133-aa1bfce9e356.png)

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.48.0
Chromium: 111
filterset (summary): 
  network: 88438
  cosmetic: 41190
  scriptlet: 17383
  html: 0
listset (total-discarded, last-updated): 
  added: 
    NLD-0: 2574-14, now
  default: 
    user-filters: 0-0, never
    easylist: 64839-48, now
    easyprivacy: 30738-1176, now
    plowe-0: 3751-934, now
    ublock-abuse: 78-0, now
    ublock-badware: 5411-114, now
    ublock-filters: 33075-113, now
    ublock-privacy: 295-2, now
    ublock-unbreak: 1939-46, now
    urlhaus-1: 6933-0, now
filterset (user): [empty]
trustedset: 
  added: [array of 76 redacted]
switchRuleset: 
  added: [array of 3 redacted]
modifiedUserSettings: [none]
modifiedHiddenSettings: [none]
supportStats: 
  allReadyAfter: 2746 ms (selfie)
  maxAssetCacheWait: 78 ms
popupPanel: 
  blocked: 28
  blockedDetails: 
    goplay.be: 1
    cxense.com: 1
    doubleclick.net: 11
    gemius.pl: 2
    google-analytics.com: 1
    google.com: 2
    googlesyndication.com: 3
    googletagmanager.com: 1
    gstatic.com: 3
    imasdk.googleapis.com: 1
    onthe.io: 1
    pbstck.com: 1
```
</details>

**Comment by Maiskolf** [comment]
Why was this issue closed?

**Comment by mapx-** [comment]
- duplicate
- did you report at NLD list too ? They are able to use native ip.

**Comment by Maiskolf** [comment]
I created a new issue since the previous issue (https://github.com/uBlockOrigin/uAssets/issues/16881) was related to ads, yet this is related to the remaining ad layover.  IMO that's not a duplicate, especially since the ad layover wasn't resolved in the previous issue?

Previous goplay.be issues have been fixed in this repository (e.g. https://github.com/uBlockOrigin/uAssets/issues/11674) so I didn't know I have to report these issues somewhere else. Am I correct in assuming I'll need to report the issue here?: https://github.com/EasyDutch-uBO/EasyDutch/issues
